lib/common/constants.js

```diff
@@ -26,7 +26,7 @@ function constantsFactory (path) {
               error: 'red',
               warning: 'yellow',
               info: 'green',
-              debug: 'blue',
+              debug: 'cyan',
```

Comment:
For my eye, I just see the green color in terminal with dark background is not so visible, but the color "cyan" is comfortable in both dark and light background, so that's why I change this.


Felix

-----Original Message-----
From: Ben Broderick Phillips [notifications@github.com]
Received: 星期六, 30 1月 2016, 2:41
To: RackHD/on-core [on-core@noreply.github.com]
CC: Yuan, Felix [Felix.Yuan@emc.com]
Subject: Re: [on-core] improve log output (color/level/caller) (#66)


In lib/common/constants.js<https://github.com/RackHD/on-core/pull/66#discussion_r51297728>:

> @@ -26,7 +26,7 @@ function constantsFactory (path) {
>                error: 'red',
>                warning: 'yellow',
>                info: 'green',
> -              debug: 'blue',
> +              debug: 'cyan',


Heh, so, I can tell the difference between cyan and the other colors, however I did notice that I actually can't tell the difference between info and warning messages (green and yellow)!

—
Reply to this email directly or view it on GitHub<https://github.com/RackHD/on-core/pull/66/files#r51297728>.